lisp_ease/src/queries/designators.lisp

```diff
@@ -0,0 +1,144 @@
+(in-package :le)
+
+;; --- DESIG FACT GROUP --------------------------------------------------------------------
+;; the fact groups were taken from Gayane Kazhoyan's "mobile-pick-and-place-plans" and were adapted to the needs of this thesis. (link to original: https://github.com/cram2/cram/blob/master/cram_common/cram_mobile_pick_place_plans/src/pick-place-designators.lisp)
```

Comment:
Please make sure the lines are max 80 (in the worst case 90, in the worst worst case 95) characters long.